sdk/core/core-tracing/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0-preview.8 (Unreleased)
+
+- Fix tracingPolicy() to set standard span attributes [PR link](https://github.com/Azure/azure-sdk-for-js/pull/6565).  Now the following are set correctly for the spans
```

Comment:
This change was done in `core-http`, so the changelog there should reflect this right?
The only change in `core-tracing` that was made as part of https://github.com/Azure/azure-sdk-for-js/pull/6565 was related to TestSpan

Reply:
yeah that's right. will update core-http CHANGELOG